src/registrar/tests/test_admin.py

```diff
@@ -367,3 +378,209 @@ def test_get_fieldsets_non_superuser(self):
 
     def tearDown(self):
         User.objects.all().delete()
+
+
+class AuditedAdminTest(TestCase):
+    def setUp(self):
+        self.site = AdminSite()
+        self.factory = RequestFactory()
+        self.client = Client(HTTP_HOST="localhost:8080")
+
+    def order_by_desired_field_helper(
+        self, obj_to_sort: AuditedAdmin, request, field_name, *obj_names
+    ):
+        formatted_sort_fields = []
+        for obj in obj_names:
+            formatted_sort_fields.append("{}__{}".format(field_name, obj))
+
+        # Not really a fan of how this looks, but as the linter demands...
```

Comment:
What was linter complaining about?

Reply:
Eh just length again, probably good to remove that comment there